Source/Fuse.Reactive.Bindings/Instance.API.uno

```diff
@@ -0,0 +1,356 @@
+
+			Typically, this collection is not referred to directly. Rather, it will contain all of the children of the `Each` tag in UX.
+		*/
+		[UXPrimary]
+		public IList<Template> Templates
+		{
+			get
+			{
+				if (_templates != null)
+					return _templates;
+					
+				_rootTemplates = new RootableList<Template>();
+				if (IsRootingCompleted)
+					_rootTemplates.Subscribe(OnTemplatesChanged, OnTemplatesChanged);
+				_templates = _rootTemplates;
+				return _templates;
+			}
+		}
+
+		InstanceDefer _defer = InstanceDefer.Frame;
+		/** Defers the creation items to avoid processing bottlenecks.
+		
+			The default is `Frame`.
+		*/
+		public InstanceDefer Defer
+		{
+			get { return _defer; }
+			set { _defer = value; }
+		}
+		
+		InstanceReuse _reuse = InstanceReuse.None;
+		/** Attempts to reuse template instances when items are being removed and created.
+		
+			The default is `None`
+			
+			Be aware that when using this feature several other features may no longer work as expected, such as:
+				- RemovingAnimation: the reused items are not actually removed
+				- AddingAnimation: the resused items are not actually added, just moved
+				- Completed: As a reused item is not added/removed it will not trigger a second time
+				
+			This feature will remain experimental until we can figure out which of these issues can be solved, avoided, or just need to be accepted.
+				
+			@experimental
+		*/
+		public InstanceReuse Reuse
+		{
+			get { return _reuse; }
+			set { _reuse = value; }
+		}
+		
+		InstanceObjectMatch _objectMatch = InstanceObjectMatch.None;
+		/**
+			Reuses existing nodes if the new objects match the old ones.
+			
+			This field is typically set implicity. It defaults to `None`. Use `ObjectId` instead if you want to match based on a id field. 
+			
+			If you need to match on the observable value itself, set this to `Object`, otherwise it works like `ObjectId`
+			
+			@see ObjectId
+		*/
+		public InstanceObjectMatch ObjectMatch
+		{
+			get { return _objectMatch; }
+			set { _objectMatch = value; }
+		}
+		
+		string _objectId = null;
+		/**
+			If specified will reuse existing items if a new item is created that has the same id.
+			
+			The `ObjectId` is a key into the provided objects. If the key is not found the item will not have an id, and will not be matched.
+			
+			Matched items keep the same Node instances that they had before. This makes it suitable for using in combination with `LayoutAnimation`. It also makes it possible to use `AddingAnimation` and `RemovingAnimation` with `Each`, as the Node lifetime will now follow the logical lifetime.
+			
+			This feature works in conjunction with `replaceAt` and `replaceAll` on Observable's.
+			
+			NOTE: This feature, if using animations, does not yet operate well in combination with `Reuse`. It may result in reuse of unintended items and/or unexpected animations.
+			https://github.com/fusetools/fuselibs-public/issues/175
+		*/
+		public string ObjectId
```

Comment:
The name of this property is confusing. Sounds to me like `ObjectKey` or `IdentityKey` is a better name, matching the convention of `MatchKey` and `TemplateKey`.